SetupScript.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+###base script to fresh install systems onto base image for raspberry pi
+
+#configure displays
+cd ~
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+cd Epsilon-Raspberry
+cd primary/
+sudo cp xorg.conf /etc/X11/
```

Comment:
`sudo` hurts the portability and ability to automate these scripts. It's better to require sudo to run the script.